src/components/layout.css

```diff
@@ -1,3 +1,7 @@
 html.iframe body {
     background-color: transparent !important;
 }
+
+html.dark .list-group-item a {
+  color: #e1e1e1 !important;
+}
```

Comment:
You may want to configure your editor to add trailing newlines, it makes git cranky when files don't have them :)

Reply:
Ah, I didn't know that. I have edited my settings accordingly. Thanks!